Configuration.props

```diff
@@ -151,8 +155,33 @@
     <AndroidSupportedTargetJitAbisForConditionalChecks Condition=" !$(AndroidSupportedTargetJitAbisForConditionalChecks.StartsWith (':')) " >:$(AndroidSupportedTargetJitAbisForConditionalChecks)</AndroidSupportedTargetJitAbisForConditionalChecks>
     <AndroidSupportedTargetJitAbisSplit>$(AndroidSupportedTargetJitAbis.Split(':'))</AndroidSupportedTargetJitAbisSplit>
   </PropertyGroup>
+
   <ItemGroup>
-    <AndroidSupportedTargetJitAbi Include="$(AndroidSupportedTargetJitAbisSplit)" />
+    <!-- It appears we cannot use batching outside a <Target> so we cannot use a set of items specifying the necessary API level to generate the
+         AndroidSupportedTargetJitAbi items with appropriate metadata. Thus we need to do it manually as below -->
+    <AndroidSupportedTargetJitAbi
+        Include="armeabi-v7a"
+        Condition=" $(AndroidSupportedTargetJitAbis.Contains ('armeabi-v7a')) ">
```

Comment:
This should be `$(AndroidSupportedTargetJitAbisForConditionalChecks.Contains (':armeabi-v7a:'))`, along with the other locations in this file.

The rationale *now* is for "sanity"/"future-proofing";  historically, the issue was `armeabi`, as `AndroidSupportedTargetJitAbis.Contains('armeabi')` would match *both* `armeabi` and `armeabi-v7a`.  We thus want/need to "wrap" the values in `:` to ensure that we don't get inadvertent substring mismatches.

This is still an issue for `x86`, as `x86` is a substring of `x86_64`.